### ISSUE TITLE
 Gutenberg: v10.1.x plugin upgrade

### ISSUE THREAD

**Issue by WunderBart** [Automattic/wp-calypso#50323]
Previous upgrade: https://github.com/Automattic/wp-calypso/issues/50060
Release notes: [v10.1.0](https://github.com/WordPress/gutenberg/releases/tag/v10.1.0), [10.1.1](https://href.li/?https://github.com/WordPress/gutenberg/releases/tag/v10.1.1)

### Process

- [x] Install and activate
  - [x] Copy & install v10.1.0-rc.1 (D57602-code, D57605-code)
  - [x] Activate v10.1.0-rc.1 on edge (D57608-code)
  - [x] Copy & install v10.1.0 (D58006-code, D58008-code)
  - [x] Activate v10.1.0 on edge (D58009-code)
  - [x] Activate v10.1.0 on production (D57609-code)
  - [x] Request update for Atomic sites (p9o2xV-1hM-p2)
  - [x] Copy & install v10.1.1 (D58202-code, D58203-code)
  - [x] Activate v10.1.1 on edge (D58205-code)
  - [x] Activate v10.1.1 on production (D58206-code)
  - [x] Request update for Atomic sites (p9o2xV-1id-p2)
- [x] Testing
  - [x] Calypso E2E (https://github.com/Automattic/wp-calypso/pull/50447)
  - [x] Team City (D57609-code)
- [x] Publish internal announcements
- [x] Clean up unused releases
  - [x] v9.9.3 (D58139-code)
  - [x] v10.1.0-rc.1 (D58141-code)
- [x] Update the Gutenberg Upgrade Log [pcoGjb-g-p2] and list patches applied
- [x] Open a new issue for the next upgrade, transfer remaining tasks, close this issue.

### Issues / Blockers 🐛 

- [x] Block's focus stealing regression 
  - Discovered via E2E: https://github.com/Automattic/wp-calypso/pull/50447#issuecomment-785960184
  - Reported in: https://github.com/WordPress/gutenberg/issues/28932#issuecomment-785828656
  - Upstream fix: https://github.com/WordPress/gutenberg/pull/29342 (will be included in v10.1.0)
  - Slack context: p1614255122109100-slack-C02A41GCS

- [ ] Classic: Infinite content duplication loop when wrapping with anchor tag [#29338](https://github.com/WordPress/gutenberg/issues/29338) (not a blocker)
  - Reported in: p1613735197155300/1613458151.107400-slack-CBTN58FTJ 

- [ ] Reusable block can't be broken down into individual blocks, no specific root cause identified yet. Seems specific to using some blocks inside the reusable block, though it's not known yet at this time. (edge case / not a blocker)
  - Reported in: p1614866772382300-slack-C7YPUHBB2

- [x] Crashing editor/browser when latest Instagram posts block is added (this would probably be considered a blocker for the release, but unfortunately we only found it after releasing. A talk about regression test starts here: p1614903492124800/1614895798.108900-slack-CBTN58FTJ).
  - Reported in: https://github.com/Automattic/wp-calypso/issues/50778
  - Slack context: p1614895798108900-slack-CBTN58FTJ
  - JP fix: https://github.com/Automattic/jetpack/pull/19034;
  - Gutenberg fixes: 
    - https://github.com/WordPress/gutenberg/pull/29491;
    - https://github.com/WordPress/gutenberg/pull/29582.

---
FYI: @Automattic/team-calypso

**Comment by fullofcaffeine** [comment]
This is done 🎉! Next one: https://github.com/Automattic/wp-calypso/issues/50775.

**Comment by fullofcaffeine** [comment]
Re-opening due to https://github.com/Automattic/wp-calypso/issues/50778 (slack context: p1614895798108900-slack-CBTN58FTJ). A patch-release including the fix has been released already. I'll close once the update is done.
